servicetalk-transport-netty-internal/build.gradle

```diff
@@ -42,6 +42,10 @@ dependencies {
   implementation "io.netty.incubator:netty-incubator-transport-native-io_uring:$nettyIoUringVersion"
   runtimeOnly "io.netty.incubator:netty-incubator-transport-native-io_uring:$nettyIoUringVersion:linux-x86_64"
   runtimeOnly "io.netty.incubator:netty-incubator-transport-native-io_uring:$nettyIoUringVersion:linux-aarch_64"
+  runtimeOnly( group:"io.netty", name:"netty-tcnative-boringssl-static", classifier:"linux-x86_64")
```

Comment:
Note as of tcnative-2.0.49 Netty [no longer repackages into an uber jar](https://github.com/netty/netty-tcnative/commit/6fb5b01208671c007ad7f752e51ccc8b25dfca2f), so build tools will be able to dedup between  the uber jar and individual classifier artifacts.

thx @normanmaurer ❤️